### PR TITLE
Fixes issues found during code reviews for the mint-nft example

### PR DESCRIPTION
**Pull request by gbibeaul**
Fixes the following on the NFT mint:

- Replaces a button without onClick by a loading state in the upload zone
- Improves the readme and corrects some values (better domain)
- changes the package.json package name
- removes left over unused pages and dom elements

### CODE REVIEW

File: solutions/mint-nft/README.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # mint-nft example
 
-This example shows how -------------------------
+This example shows how to mint an NFT using Next.js and [Moralis](https://moralis.io/)
 
 ## Demo
 
-https://mint-nft.vercel.app
+https://mint-nft-kappa.vercel.app/
```
Comment (lfades) [inline solutions/mint-nft/README.md:7]: We need a better example name, e.g: `solutions-mint-nft`, and also remove the trailing slash.

File: solutions/mint-nft/components/UploadNft.tsx [hunk 2]
```diff
@@ -96,11 +105,7 @@ export const UploadNft: React.VFC<Props> = ({ onDone }) => {
             {imageWarning}
           </aside>
         )}
-        <div className="mt-6 sm:items-center flex justify-center">
-          <Button disabled={disabled} size="lg" variant="black">
-            {loading ? <LoadingDots /> : 'Confirm'}
-          </Button>
-        </div>
+        <div className="mt-6 sm:items-center flex justify-center"></div>
```
Comment (lfades) [inline solutions/mint-nft/components/UploadNft.tsx:108]: What does this empty div do?

**Review by lfades — CHANGES_REQUESTED** [review]
Added some comments, also `package-lock.json` shouldn't be removed.

File: solutions/mint-nft/README.md (hunk 1 above)
Comment (gbibeaul) [reply solutions/mint-nft/README.md]: Good point! I claimes solutions-mint-nft and removed the slash 👍

File: solutions/mint-nft/components/UploadNft.tsx (hunk 2 above)
Comment (gbibeaul) [reply solutions/mint-nft/components/UploadNft.tsx]: Nada 🤡 removed

**Comment by lfades** [comment]
Updated `package-lock.json` to see if the example deploys successfully.

**Pull request merged**